py-requirements.txt

```diff
@@ -4,3 +4,4 @@ pytest-cov
 flake8~=5.0.0 # MU_CHANGE - update to 5.0.0 or later
 pyopenssl
 pefile
+cx_Freeze

```

Comment:
can we specify a version of cxfreeze?

Reply:
Good suggestion. I think we've leveled out on using versions and then let dependabot take it over.

Reply:
I ended up not including this change since it really is not a requirement for normal users. Also the pipeline is failing due to one of the pip dependencies was somehow still using legacy installation.